Week3/HW3/1_strings.js

```diff
@@ -0,0 +1,6 @@
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+console.log(myString);
+// g is for global if there is no g, it will replace only the first one,
+// if we do not want to use g, then we have to make a loop.
+newString = myString.replace(/,/g, " "); 
+console.log(newString);
```

Comment:
Two things here:
1. Since we aren't reassigning `myString` anywhere else in the code, it is better to use `const` instead of `let` while creating the variable.
2. You didn't use `var`, `let` or `const`(preferred) while creating the `newString` variable. 🙈

Reply:
Hi, @yash-kapila.
Thank you very much for reviewing my amateur work. 
I implemented and understood the necessity and role of const better now.